### PR TITLE
Remove "oil&gas"

### LINKED ISSUE #27: ald_demo has sector "oil&gas" whereas all *classifications have "oil and gas"
the "oil and gas" sector is mislabelled as "oil&gas" in the `ald_demo`, causing `r2dii.match::match_name()` to lack any matches for this sector if `by_sector=T`. 

TODO: 
- [x] remove refs to "oil&gas" (#29)
- [ ] write test (2DegreesInvesting/r2dii.match#99) to ensure all sectors are represented in demo match_name output

### PR DESCRIPTION
**Pull request by jdhoffa**
Closes #27

### CODE REVIEW

File: .Rproj.user/3A98825C/pcs/source-pane.pper [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (maurolepore) [inline .Rproj.user/3A98825C/pcs/source-pane.pper:1]: Oops, it seems that .Rproj.user files are not properly ignored. When you see them staged, you may do:

```
git rm --cached -r .
git add .
git update-index --assume-unchanged
git commit -a -m "forget cached"
```

https://github.com/forestgeo/learn/issues/5

File: data-raw/ald_demo.csv [hunk 2]
```diff
@@ -68,19 +68,19 @@ affinity renewables inc.,power,renewablescap,MW,2037,20874.066126101905,NA,AS,MW
...
-africa oil corp,oil&gas,oil,GJ per day,2017,30413476.922810085,3326814.9054234186,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2018,29911333.490360737,2967066.6686623553,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2019,29409190.05791139,2607318.431901292,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2020,28907046.62546204,2247570.1951402286,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2021,28404903.193012692,1887821.9583791657,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2022,27902759.760563344,1528073.7216181024,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2023,27400616.328113995,1168325.484857039,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2024,26898472.89566465,808577.2480959762,CD,CM,1,TRUE,TRUE,1994Q1
-africa oil corp,oil&gas,oil,GJ per day,2025,26396329.463215303,448829.01133491244,CD,CM,1,TRUE,TRUE,1994Q1
+africa oil corp,oil and gas,oil,GJ per day,2013,32422050.652607474,4765807.852467671,CD,CM,1,TRUE,TRUE,1994Q1
```
Comment (maurolepore) [inline data-raw/ald_demo.csv:71]: Thanks!
We'll also need to update data/ald_demo by sourcing data-raw/ald_demo.R.
That should make the regression test to fail. So we'll also need to temporarily set `update = TRUE`, then run tests twice and set `update = FALSE` back again.

**Comment by maurolepore** [comment]
Ups, sorry I didn't realize I was commenting on a closed PR

File: data-raw/ald_demo.csv (hunk 2 above)
Comment (jdhoffa) [reply data-raw/ald_demo.csv:71]: i knew i forgot something! havent looked at dataraw in a while haha

**Pull request closed without merging**